src/domUpdates.js

```diff
@@ -148,8 +150,13 @@ let domUpdates = {
   },
 
   displayBookedMessage() {
-    greeting.innerHTML = "THANKS FOR BOOKING WITH US!"
-    availableRoomsSection.innerHTML += ''
+    greeting.innerHTML = "Thanks for booking with us!"
+    availableRoomsSection.innerHTML = `<img class="large-logo large-logo-js" src="images/natural-beauty.png" alt="large hood overlook hotel logo"/>`
```

Comment:
It's much easier to see what's happening with the cards if I simply 'replace' that entire container with the 'thank you'. (It's on a setTimeout so it comes back after a few seconds.)